static/js/composebox_typeahead.js

```diff
@@ -745,6 +767,8 @@ exports.content_highlighter = function (item) {
         return typeahead_helper.render_typeahead_item({
             primary: item.text,
         });
+    } else if (this.completing === 'stream' && typeof(item) === 'string') {
```

Comment:
I don't like this if here. It works but it's confusing for someone who will edit this code in the future and also can become a source of bug IMO. But I also think that it's a little bit of 'overkill' to encapsulate the string in an object just to indicate that it's a topic suggestion in the '#' query.

Does anyone have a suggestion about what could I do here?

Reply:
Maybe add a comment to clarify for future developers?